docs/standard/serialization/binary-serialization.md

```diff
@@ -85,10 +131,42 @@ As the nature of binary serialization allows the modification of private members
 - <xref:System.Data.SqlTypes.SqlInt16?displayProperty=nameWithType>   
 - <xref:System.Data.SqlTypes.SqlInt32?displayProperty=nameWithType>   
 - <xref:System.Data.SqlTypes.SqlInt64?displayProperty=nameWithType>   
+- <xref:System.Data.SqlTypes.SqlNotFilledException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Data.SqlTypes.SqlNullValueException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
 - <xref:System.Data.SqlTypes.SqlString?displayProperty=nameWithType>   
+- <xref:System.Data.SqlTypes.SqlTruncateException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Data.SqlTypes.SqlTypeException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Data.StrongTypingException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Data.SyntaxErrorException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Data.VersionNotFoundException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.DataMisalignedException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
 - <xref:System.DateTime?displayProperty=nameWithType>   
 - <xref:System.DateTimeOffset?displayProperty=nameWithType>   
 - <xref:System.Decimal?displayProperty=nameWithType>   
+- <xref:System.Diagnostics.Contract.ContractException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
```

Comment:
Note that the namespace is incorrect (Contracts, not Contract). But even if that were corrected, the link won't resolve, since we haven't reflected on the type. Can you fence it and add a comment, like this:
\`System.Diagnostics.Contracts.ContractException\` \<!--zz \<xref:System.Diagnostics.Contracts.ContractException?displayProperty=fullName> -->